### ISSUE TITLE
MNIST training data from yann lecun is not available (503 Service Unavailable)

### ISSUE THREAD

**Issue by YeonghyeonKO** [kumar-shridhar/PyTorch-BayesianCNN#62]
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/46114393/111263122-acdfc900-8668-11eb-86ed-78e76519eded.png">

When I tried to run uncertainty_estimation.py, it had kept returned 503 Service Unavailable error.

Since the link for downloading `mnist/train-labels-idx1-ubyte.gz` is unavailable now, would you change the code not to access [here](http://yann.lecun.com/exdb/mnist/train-labels-idx1-ubyte.gz) ? (Even I downloaded notMNIST_small.tar and put it into `/data`, the codes tries to search the link so returns 503 error.

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/46114393/111263190-c97c0100-8668-11eb-8588-f1d823af7470.png">
